src/mig/modules/agentdestroy/agentdestroy.go

```diff
@@ -22,7 +22,7 @@ import (
 func init() {
 	mig.RegisterModule("agentdestroy", func() interface{} {
 		return new(Runner)
-	})
+	}, false)
```

Comment:
I'm guessing these are all still to be converted since RegisterModule is no more?